src/components/Carousel.tsx

```diff
@@ -1,18 +1,186 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './Carousel.scss';
 
-const Carousel: React.FC = () => (
-  <div className="Carousel">
-    <ul className="Carousel__list">
-      <li><img src="./img/1.png" alt="1" /></li>
-      <li><img src="./img/1.png" alt="2" /></li>
-      <li><img src="./img/1.png" alt="3" /></li>
-      <li><img src="./img/1.png" alt="4" /></li>
-    </ul>
-
-    <button type="button">Prev</button>
-    <button type="button">Next</button>
-  </div>
-);
+interface Props {
+  images: string[];
+}
+
+const Carousel: React.FC<Props> = ({ images }) => {
+  const [translate, setTranslate] = useState(0);
+  const [itemWidth, setItemWidth] = useState(130);
+  const [frameSize, setFrameSize] = useState(3);
+  const [step, setStep] = useState(3);
+  const [animationDuration, setAnimationDuration] = useState(1000);
+  const [infinityMode, setInfinityMode] = useState(false);
```

Comment:
suggest to make one state for all inputs as object and use event.target.name for change proper field

```suggestion
const [inputs, setInputs] = useState({
  itemWidth: 0,
  ...
});
```
also caruousel shouldn't to know about inputs and except just images and values of inputs.

![Screenshot from 2023-09-09 16-16-42](https://github.com/mate-academy/react_carousel/assets/33029864/b74eea77-17a1-4a8b-803b-02aef9601d9c)

move inputs and state in App component or create separate component, but it should be in App component.